code/modules/modular_computers/file_system/programs/generic/news_browser.dm

```diff
@@ -3,6 +3,7 @@
 	filedesc = "NTNet/ExoNet News Browser"
 	extended_desc = "This program may be used to view and download news articles from the network."
 	program_icon_state = "generic"
+	usage_flags = PROGRAM_CONSOLE | PROGRAM_LAPTOP | PROGRAM_TABLET | PROGRAM_STATIONBOUND | PROGRAM_WRISTBOUND
```

Comment:
pAI can't access news? Just do not add restrictions.

Reply:
Already addressed in my review.